Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -66,6 +66,13 @@ public class RobotContainer
   private double                               MaxAngularRate  = 1.5 * Math.PI; // 3/4 of a rotation per second max angular velocity
   private Command                              m_autoCommand;
 
+  //Poses for limelight paths
```

Comment:
These should probably be constants in the Constants.java file with their own class. Maybe PATHConsts or something similar. This should also include a "constraints" constant that is fed into pathfindToPose.